python/pyarrow/conftest.py

```diff
@@ -76,14 +80,21 @@
     'pandas': False,
     'parquet': False,
     'parquet_encryption': False,
+    'processes': True,
     'requires_testing_data': True,
     's3': False,
     'slow': False,
     'snappy': Codec.is_available('snappy'),
     'substrait': False,
+    'threading': is_threading_enabled(),
     'zstd': Codec.is_available('zstd'),
 }
 
+if sys.platform == "emscripten":
+    # emscripten doesn't support multiprocessing or gdb
```

Comment:
```suggestion
    # emscripten doesn't support multiprocessing/subprocess or gdb
```

From seeing the tests that are marked with `processes`, it's more the use of `subprocess` module than `multiprocessing`? (which in practice has the same effect of spawning multiple processes, but I found the mention of multiprocessing (thinking about the stdlib module) here confusing for a moment)